setup.cfg

```diff
@@ -0,0 +1,12 @@
+[wheel]
+universal = 1
+
+[isort]
+atomic=true
+multi_line_output=5
+known_standard_library=types
+known_third_party=pytest
+known_first_party=apistar
```

Comment:
Do we need this `isort` config. How do things differ if we dropped it, and just used the standard? Is it possible to enforce a "everything in one section, alphabetically" policy, rather than the standard "builtin, third-party, first party" sections?